iOS/Data/Model/Video+Download.swift

```diff
@@ -90,6 +90,70 @@ extension Video {
         return nil
     }
 
+    @available(iOS 13, *)
+    var downloadStreamAction: UIAction? {
```

Comment:
The newly added code is almost the same as the code of `streamUserAction` and `slidesUserAction`. Instead of having the code duplicated once for `UIAlertAction` and `UIAction`, I suggest doing it once for an intermediary type `Action`.
```
struct Action {
    let title: String
    let image: UIImage
    let actionHandler: ((UIAlertAction) -> Void)?
} 
```
Then all available actions can be combined. Have a look at the existing computed property `userActions`.  The combined actions can then be mapped to either  `[UIAlertAction]` and `[UIAction]`.

PS: Don't forget about the action which starts the download for the stream and the slides at the same time.